packages/eslint-plugin-openshift/rule-overrides/react-typescript-prettier.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Nit: please use 2 spaces for indentation, this is part of EditorConfig we use.

I recommend using VS Code since we already have `.vscode/settings.json` config in the repo.

Also, I'd recommend installing the relevant VS Code extensions to take full advantage of ^^ config:

```sh
code --install-extension EditorConfig.EditorConfig
code --install-extension dbaeumer.vscode-eslint
code --install-extension esbenp.prettier-vscode
```

Reply:
added them, thank you :-)